src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -230,17 +228,12 @@ This command will accept a key with either 16 or 32 characters.
 
 ## Widevine Encryption
 
-Everything in this command except the name of your files and the `--content_id`
-flag should be copied exactly from the example. The `--content_id` is 16 or 32
-random hex digits. Use the keys provided here instead of your own. (This is how
-Widevine works.)
- 
     packager \
-      input=glocken.mp4,stream=video,output=enc_glocken.mp4 \
-      input=myvide.mp4,stream=audio,output=enc_myaudio.m4a \
+      input=glocken.mp4,stream=video,output=enc_video.mp4 \
+      input=glocken.mp4,stream=audio,output=enc_audio.m4a \
       --enable_widevine_encryption \
       --key_server_url "https://license.uat.widevine.com/cenc/getcontentkey/widevine_test" \
-      --content_id "fd385d9f9a14bb09" --signer "widevine_test" \
+      --content_id "16_Rand_Hex_Chrs" --signer "widevine_test" \
```

Comment:
'content_id' is an unique identifier identifies the content. There is no restriction on the size of content id. In fact, you can use an arbitrary size string as content_id. It does need to be hex converted before supplying it to packager.

It does not need to be randomly generated either.